src/ApiService/ApiService/OneFuzzTypes/Model.cs

```diff
@@ -918,26 +906,23 @@ public record SecretAddress<T>(Uri Url) : ISecret<T> {
 public record SecretData<T>(ISecret<T> Secret) {
 }
 
+[SkipRename]
+public enum JobResultType {
+    CoverageData,
+    RuntimeStats,
+}
+
 public record JobResult(
-    [PartitionKey][RowKey] Guid JobId,
```

Comment:
changing partition key and row key to a new one - that means that all previous metrics will be pretty much impossible to retrieve. 

Is that what you want here ? Starting from clean slate after this PR is deployed ?

Reply:
Yes. We are going to start from a clean slate. Data is already only kept for 30 days right now, so we will just let data build up for thirty days or so and then switch over.